apollo/apollo.go

```diff
@@ -206,38 +197,45 @@ func (c *client) RegisterConfigCallback(param ConfigParam,
 		callback(data, c.parser)
 	}
 
-	configMap := c.acli.GetNameSpace(param.NameSpace)
+	configMap := c.acli.GetNameSpace(param.nameSpace)
```

Comment:
This comment is for the code in line 194:

The implementation of `DeregisterConfig` is not appropriate, it starts a new goroutine when registering a new config, so it should close the goroutine when deregistering.

Reply:
My previous idea was that after stopping, if goroutine continued to listen, it would send an error to errorCh, use errorCh to listen for errors, and then close goroutine

I have added the 'stop' field and used 'close' to close the stop chan in the 'DeregisterConfig' function, so that all goroutines receive a message to close it

Reply:
```
it would send an error to errorCh, use errorCh to listen for errors, and then close goroutine
```
If there is more than one goroutine to receive the `errorCh` but only one goroutine to consume the error message, I think this is not a good idea to do that. 
A client can register multiple configs, do not close the client if only one is unregistered.